system/modules/core/classes/Theme.php

```diff
@@ -641,7 +643,7 @@ protected function extractThemeFiles($arrFiles, $arrDbFields)
 	 */
 	public function exportTheme(\DataContainer $dc)
 	{
-		// Get the theme meta data
+		/** @var \DataContainer|object $dc */
```

Comment:
This should be a method property, and it should also not force a type then. That would make it easier to just pass in an object with property `id`.